src/HypothesisTests.jl

```diff
@@ -110,7 +110,7 @@ function show_params(io::IO, test::T, ident="") where T<:HypothesisTest
         lengths = [length(string(T.names[i])) for i in fieldidx]
         maxlen = maximum(lengths)
 
-        for i = 1:length(fieldidx)
+        for i in eachindex(fieldidx)
```

Comment:
One should be a bit careful here since `i` is used both for indexing into `fieldidx` and `maxlen-lengths`. Here, I think it would be easier to avoid indexing altogether and write
```suggestion
        for (fieldidx_i, maxlen-lengths_i) in zip(fieldidx, maxlen-lengths)
```
and use these variables in the body of the for-loop.

Reply:
I don't think this suggestion is correct: `maxlen-lengths` will error since the former is a scalar and the latter is an array, and `maxlen-lengths_i` is not a valid identifier. It's likely this method doesn't have tests, which is why CI didn't complain about it.

Reply:
> One should be a bit careful here since `i` is used both for indexing into `fieldidx` and `maxlen-lengths`.

That's true in general, but in this case by construction the arrays are plain `Vector`s with the same length, so indexing both with `i` should be safe.